app/src/test/java/org/simple/clinic/teleconsultlog/teleconsultrecord/screen/TeleconsultRecordEffectHandlerTest.kt

```diff
@@ -78,4 +91,45 @@ class TeleconsultRecordEffectHandlerTest {
 
     verifyZeroInteractions(uiActions)
   }
+
+  @Test
+  fun `when create teleconsult record effect is received, then create the teleconsult record`() {
+    // given
+    val patientUuid = UUID.fromString("cd8c77c4-67de-482a-bfd0-2b035291b45d")
+    val teleconsultRecord = TestData.teleconsultRecord(
+        id = uuidGenerator.v4(),
+        patientId = patientUuid,
+        medicalOfficerId = user.uuid,
+        teleconsultRecordInfo = TestData.teleconsultRecordInfo(
+            recordedAt = Instant.parse("2018-01-01T00:00:00Z"),
+            teleconsultationType = Audio,
+            patientTookMedicines = Yes,
+            patientConsented = Yes,
+            medicalOfficerNumber = null
+        ),
+        teleconsultRequestInfo = null,
+        createdAt = Instant.now(utcClock),
+        updatedAt = Instant.now(utcClock)
+    )
+
+    doNothing().whenever(teleconsultRecordRepository).createTeleconsultRecordForMedicalOfficer(
```

Comment:
Is the `doNothing()` necessary here?

Reply:
Without this mock is failing for void return methods.

Reply:
Can we recheck? If I remember correctly, Mockito automatically handles stubs for void returns.

Reply:
I have tested with this and it fails with `UnfinishedStubbingException`
```kt
    whenever(teleconsultRecordRepository.createTeleconsultRecordForMedicalOfficer(
        teleconsultRecordId = teleconsultRecord.id,
        patientUuid = teleconsultRecord.patientId,
        medicalOfficerId = teleconsultRecord.medicalOfficerId,
        teleconsultRecordInfo = teleconsultRecord.teleconsultRecordInfo!!
    ))
```

Reply:
Huh, this is new. Checking.

Reply:
Oh wait, I mean you should just skip setting up a stub at all in the first place and use a `verify(teleconsultRepository).createTeleconsultRecordForMedicalOfficer()` verification call.

Reply:
If you don't start a stubbing with `whenever`, Mockito will handle it by returning `null` for methods which return something and `Unit` for methods that don't.

Reply:
Ah okay, will do that 👍🏽

Reply:
Yup, I tested it out right now by just removing the entire stubbing and it works.